.github/workflows/ids-check.yml

```diff
@@ -0,0 +1,89 @@
+name: ids-check
+on:
+  pull_request:
+    branches: [main]
```

Comment:
This needs to be changed if you want the workflow to run on stacked PRs:

https://llvm.org/docs/CIBestPractices.html#ensuring-workflows-run-on-the-correct-events

Reply:
Yeah, we would want them on release branches as well.